test/proxy/write_to_proxies.json

```diff
@@ -0,0 +1,74 @@
+[
+    {{ range $idx, $offset := int_range 0 (datastore "max_entries") }}
+        {{ if gt $idx 0 }},{{ end }}
+        {
+            "name": "Add some JSON requests to proxy 'test'",
+            "request": {
+                "server_url": "{{ datastore "req_base_url" }}",
+                "endpoint": "proxywrite/test",
+                "method": "POST",
+                "query_params": {
+                    "some": "value"
+                },
+                "header": {
+                    "X-Some": "x-header"
+                },
+                "body": {
+                    "some": [
+                        "body", "here"
+                    ],
+                    "index": "idx_{{ $offset }}"
+                }
+            },
+            "response": {
+                "statuscode": 200,
+                "body": {
+                    "offset": {{ $offset }}
+                }
+            }
+        }
+    {{ end }}
+    ,{
+        "name": "post image to proxy 'test'",
+        "request": {
+            "server_url": "{{ datastore "req_base_url" }}",
+            "endpoint": "proxywrite/test",
+            "method": "POST",
+            "body_file": "@../_res/assets/camera.jpg",
```

Comment:
Use a local image. I really dont like this "_res" stuff. Maybe you can get rid of it